samples/net/echo_client/src/echo-client.c

```diff
@@ -103,7 +103,7 @@ static inline int init_app(void)
 
 struct net_pkt *prepare_send_pkt(struct net_app_ctx *ctx,
 				 const char *name,
-				 int *expecting_len)
+				 u16_t int *expecting_len)
```

Comment:
Here's an obvious typo.

And I'm not sure why we tried generic types like "int" for things like "u16_t", as seems to be done in many places in this PR.